modules/replicator/src/main/java/org/apache/ignite/internal/replicator/message/MultiEntryResponse.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.replicator.message;
+
+import org.apache.ignite.internal.replicator.message.container.MultiEntryContainer;
+import org.apache.ignite.network.annotations.Marshallable;
+import org.apache.ignite.network.annotations.Transferable;
+
+/**
+ * Replica response.
+ */
+@Transferable(ReplicaMessageGroup.REPLICA_MULTI_RESPONSE)
+public interface MultiEntryResponse extends ReplicaResponse {
```

Comment:
I beleive that we should remove all ReplicaRequests and Responses for now, except ReplicaRequest and ReplicaResponse itself. We will bring them back during ReplicaListener Implementation.